test/extended/machines/scale.go

```diff
@@ -25,7 +25,7 @@ import (
 const (
 	machineAPIGroup       = "machine.openshift.io"
 	machineSetOwningLabel = "machine.openshift.io/cluster-api-machineset"
-	scalingTime           = 7 * time.Minute
+	scalingTime           = 12 * time.Minute
```

Comment:
Thanks for raising the timeout limit even further, I was about to say 10 min doesn't give us a lot of room.
With my tests locally, the first time it took 9:30, and the second time 10:45.

Could you just update the commit message?

Reply:
Commit title, sorry...